evm_loader/cli/src/main.rs

```diff
@@ -66,6 +122,453 @@ fn emulate(solana_url: String, evm_loader: Pubkey, contract_id: H160, caller_id:
     }
 
     account_storage.get_used_accounts(&status, &result);
+
+    Ok(())
+}
+
+fn command_create_program_address (
+    config: &Config,
+    seed: &str,
+) -> CommandResult {
+    let strings = seed.split_whitespace().collect::<Vec<_>>();
+    let mut seeds = vec![];
+    let mut seeds_vec = vec![];
+    for s in strings {
+        seeds_vec.push(hex::decode(s).unwrap());
+    }
+    for i in &seeds_vec {seeds.push(&i[..]);}
+    let (address,nonce) = Pubkey::find_program_address(&seeds, &config.evm_loader);
+    println!("{} {}", address, nonce);
+
+    Ok(())
+}
+
+fn command_create_ether_account (
+    config: &Config,
+    ether_address: &H160,
+    lamports: u64,
+    space: u64
+) -> CommandResult {
+    let (solana_address, nonce) = Pubkey::find_program_address(&[ether_address.as_bytes()], &config.evm_loader);
+    println!("Create ethereum account {} <- {} {}", solana_address, hex::encode(ether_address), nonce);
+
+    let instruction = Instruction::new(
+            config.evm_loader,
+            &EvmInstruction::CreateAccount {lamports, space, ether: *ether_address, nonce},
+            vec![
+                AccountMeta::new(config.signer.pubkey(), true),
+                AccountMeta::new(solana_address, false),
+                AccountMeta::new_readonly(system_program::id(), false)
+            ]);
+
+    let finalize_message = Message::new(&[instruction], Some(&config.signer.pubkey()));
+    let (blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
+
+    check_account_for_fee(
+        &config.rpc_client,
+        &config.signer.pubkey(),
+        &fee_calculator,
+        &finalize_message)?;
+
+    let mut finalize_tx = Transaction::new_unsigned(finalize_message);
+
+    finalize_tx.try_sign(&[&*config.signer], blockhash)?;
+    println!("signed: {:x?}", finalize_tx);
```

Comment:
Are you sure you want to print logs into stdout? eprintln! is used anywhere else.

Reply:
Fixed